RUFAS/biophysical/animal/pen.py

```diff
@@ -697,9 +699,14 @@ def get_manure_streams(self) -> dict[str, ManureStream]:
         self._validate_general_manure_stream_proportions()
         for stream in self.manure_streams:
             general_substream_proportion = float(stream.get("stream_proportion", 1.0))
+            split_ratio = general_substream_proportion * general_stream_proportion
+            manure_stream_deposit_split = (
+                general_substream_proportion if parlor_stream_proportion is not None else split_ratio
+            )
```

Comment:
This was working for the default freestall simulation but I was having a little trouble trying to split the general stream portion of this into multiple streams to test that this further splitting among multiple general streams was working properly - it's simple math so doesn't seem like it should be an issue but just something to look into if testers are able to implement that.